pallets/ddc-verification/src/lib.rs

```diff
@@ -3942,20 +4000,8 @@ pub mod pallet {
 			T::ClusterValidator::set_last_validated_era(&cluster_id, era_id)
 		}
 
-		#[pallet::call_index(11)]
-		#[pallet::weight(<T as pallet::Config>::WeightInfo::create_billing_reports())] // todo! implement weights
-		pub fn set_cluster_to_validate(
-			origin: OriginFor<T>,
-			cluster_id: ClusterId,
-		) -> DispatchResult {
-			ensure_root(origin)?;
-			ClusterToValidate::<T>::put(cluster_id);
-
-			Ok(())
-		}
-
 		// todo! Need to remove this
-		#[pallet::call_index(12)]
+		#[pallet::call_index(11)]
```

Comment:
Are you sure about the call index shift? Should we keep index `11` unused after the deletion?

Reply:
As far as I know, any API change to extrinsics should be reflected by increasing `transaction_version` field in the chain spec. It should force UI clients to re-upload the chain metadata with updated indexes. I will try to test locally.

Reply:
There is a good explanation in the comments to provided [answer](https://substrate.stackexchange.com/questions/8923/safe-way-to-remove-an-extrinsic).

**TLDR:** 
> There is nothing preventing you to reuse a certain extrinsic that was used in the past, but giving users some time to update on their side is a very healthy practice, I would say.

There are no users of the `ddc-verification` pallet extrinsics besides of Validators OCW. And we don't have any UI for that except the explorer that re-fetches the metadata once the `transaction_version` is updated. So it should be  safe